fastn-xtask/src/update_template.rs

```diff
@@ -1,44 +1,23 @@
-use std::env;
-use std::process::Command;
-use regex::Regex;
-use std::fs;
-
 pub fn update_template() -> fastn_core::Result<()> {
     println!("Updating template");
 
-    let current_dir = env::current_dir().map_err(|e| {
+    let template_dir = crate::helpers::find_directory(
```

Comment:
We do not use `crate`, we give the package a name, e.g., this one should be called `fastn_xtask` and the line should read `fastn_xtask:helpers::find_directory()`. You will have to add following to `fastn-xtask/src/lib.rs`

```rs
extern crate self as fastn_xtask;
```